optimum/exporters/tasks.py

```diff
@@ -1718,6 +1718,8 @@ def standardize_model_attributes(
             library_name (`Optional[str]`, *optional*)::
                 The library name of the model. Can be any of "transformers", "timm", "diffusers", "sentence_transformers".
         """
+        # TODO: make model_name_or_path an optional argument here.
```

Comment:
@mht-sharma Is this argument needed? For timm, the config indeed seem not to be attached to the `timm.models.xxx.Xxx` (e.g. `timm.models.resnet.ResNet`) instance, but is this an issue for the export? Do we need any info from the config.json?

For diffusers, sentence-transformers, we are fine.

Reply:
@fxmarty indeed its not required, its mapped to the `pretrained_cfg`. Created a PR with the changes in respective locations. https://github.com/huggingface/optimum/pull/1686. Feel free to merge after review

Reply:
Thank you @mht-sharma, can you in https://github.com/huggingface/optimum/pull/1686 make it so that `standardize_model_attributes` is called here: https://github.com/huggingface/optimum/blob/32a51afb7f8255f36546012b3b6bf060110e59bb/optimum/exporters/onnx/convert.py#L991

Reply:
Added